dev/archery/archery/docker/cli.py

```diff
@@ -289,3 +289,26 @@ def docker_compose_images(obj):
     click.echo('Available images:')
     for image in compose.images():
         click.echo(f' - {image}')
+
+
+@docker.command('info')
+@click.argument('service_name')
+@click.option('--only', '-o', required=False,
+              help="Show only specific docker-compose key. Examples of keys:"
+                   " command, environment, build, dockerfile")
+@click.pass_obj
+def docker_compose_info(obj, service_name, only):
+    """Show docker-compose definition info for service_name.
+
+    SERVICE_NAME is the name of the docker service defined on
+    the docker-compose. Look at `archery docker images` output for names.
+    """
+    compose = obj['compose']
+    try:
+        service = compose.config.raw_config["services"][service_name]
+    except KeyError:
+        click.echo(f'Service name {service_name} could not be found')
```

Comment:
This should behave more as an error, so for example:
```suggestion
        click.echo(f'Service name {service_name} could not be found', err=True)
        sys.exit(1)
```